Tasks/Common/azure-arm-rest/Tests/package-lock.json

```diff
@@ -92,14 +92,14 @@
       "integrity": "sha1-EPIC5MwnYQLWDibjGfl9aNoJvpw=",
       "dev": true,
       "requires": {
-        "chai": "3.5.0",
-        "debug": "2.6.9",
-        "deep-equal": "1.0.1",
-        "json-stringify-safe": "5.0.1",
-        "lodash": "4.17.5",
-        "mkdirp": "0.5.1",
+        "chai": ">=1.9.2 <4.0.0",
+        "debug": "^2.2.0",
+        "deep-equal": "^1.0.0",
+        "json-stringify-safe": "^5.0.1",
```

Comment:
There are new dependent modules. Are they scanned in our tool. You can talk to @vincentdass about this.